lib/docker.js

```diff
@@ -130,14 +130,26 @@ function createDockerFile() {
     }
 
     contents = 'FROM ' + baseImg + '\n' +
-        'RUN apt-get update && apt-get install -y ' + extraPkgs.join(' ') +
+        'RUN apt-get update --fix-missing && apt-get install -y ' + extraPkgs.join(' ') +
```

Comment:
Hm, when I was running it locally for the first time (without cache), I've got errors on this.